test/button/button-icon.visual-diff.js

```diff
@@ -35,7 +35,9 @@ describe('d2l-button-icon', function() {
 		});
 
 		it('focus', async function() {
-			await focus(page, '#normal');
+			await page.evaluate(() => D2L.TestHelpers.focus(
```

Comment:
I could further bake this into the visual-diff library (which I plan to move shortly).  If I did, this code would become:
```
await visualDiff.focus( page, '#normal', { waitFor: { selector: ['#normal', 'button'], eventName: 'transitionend' } });
```
The only thing that really does is avoids the consumer from having to use `page.evaluate` directly.

Reply:
Another API approach could be to make the user provide the element.  To get them, the consumer could use JS path collected from Chome tools.
```
await page.evaluate(() => D2L.TestHelpers.focus(
	document.querySelector('#normal'), { 
	waitFor: { 
		elem: document.querySelector('#normal').shadowRoot.querySelector('button'), 
		eventName: 'transitionend' 
	} 
});
```
But then I might argue that we're really not saving much at all.